src/System.Diagnostics.Tools/src/System/CodeDom/Compiler/GeneratedCodeAttribute.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace System.CodeDom.Compiler
+{
+    [AttributeUsage(AttributeTargets.All, Inherited = false, AllowMultiple = false)]
+    public sealed class GeneratedCodeAttribute : Attribute
+    {
+        private readonly string _tool;
+        private readonly string _vversion;
```

Comment:
(Should be _version)

I fixed this in another checkin, but it hasn't been added into the PR yet.